IMS/IMS/Controllers/InventoryController.cs

```diff
@@ -13,11 +13,62 @@ public InventoryController(IInventoryRepository InventoryRepository)
         {
             _InventoryRepository = InventoryRepository;
```

Comment:
rename inventory repository parameter to match .NET naming standards:
```suggestion
            _inventoryRepository = inventoryRepository;
```